omnigan/tutils.py

```diff
@@ -157,20 +157,26 @@ def show_tanh_tensor(tensor):
     skimage.io.imshow(image)
 
 
-def get_normalized_depth_t(arr, domain):
+def norm_tensor(t):
+    t = t - torch.min(t)
+    t /= torch.max(t)
```

Comment:
Please correct me if I am wrong. I think min - max normalization should be: (t - min) / (max - min)

Reply:
see comment above